tests/jvm/src/test/resources/metacp.expect

```diff
@@ -38,26 +67,36 @@ Schema => SemanticDB v3
 Uri => example/Acc.class
 Text => empty
 Language => Scala
-Symbols => 9 entries
+Symbols => 11 entries
 
 Symbols:
+_root_. => package _root_
+_root_.example. => package example
 _root_.example.Acc# => class Acc.{+8 decls}
-  extends AnyRef
-_root_.example.Acc#`<init>`(). => primaryctor <init>: (): Acc
+  extends scala.this.AnyRef
+_root_.example.Acc#`<init>`(). => primaryctor <init>: (): example.this.Acc
+  example => _root_.example.
   Acc => _root_.example.Acc#
-_root_.example.Acc#m1(). => private def m1: : Nothing
+_root_.example.Acc#m1(). => private def m1: : scala.this.Nothing
```

Comment:
Is this change necessary? I prefer the old type

Reply:
`scala.this.Int` is not even valid unless `scala` is an enclosing class

Reply:
See https://github.com/scalameta/scalameta/issues/1333 for discussion.